scripts/linters/html_linter.py

```diff
@@ -110,6 +110,41 @@ def handle_starttag(self, tag, attrs):
             column_number + len(tag) + 2)
         starttag_text = self.get_starttag_text()
 
+        # Check whether there is space around attributes.
+        # An = is followed either by " or {.
```

Comment:
Where do we need `attribute={...}`?

Reply:
apologies, I assumed some attr={true} or attr={condition} exist, but then again it throws error if it isnt enclosed in double quotes. Changed it